libs/utils/utils.js

```diff
@@ -257,6 +257,9 @@ export function createTag(tag, attributes, html) {
       el.setAttribute(key, val);
     });
   }
+  if (options.parent) {
+    options.parent.append(el);
+  }
```

Comment:
I don't remember if we discussed this in the initial PR, but couldn't we go with `options.parent?.append(el);` and save a couple of bits?

Reply:
:) we sure can

Reply:
I'm on the fence but coverage will still recognize the branch, so that's good.